### PR TITLE
[4.x] Remove punctuation marks Remember username E-mail

### PR DESCRIPTION
**Pull request by tecpromotion**
### Summary of Changes

There was a request in a local translation to remove the dot at the end. Users would understand the dot as part of the username and this leads to confusion.

### Testing Instructions

.

### Actual result BEFORE applying this Pull Request

COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Hello,\n\nA username reminder has been requested for your {SITENAME} account.\n\nYour username is {USERNAME}.\n\nTo login to your account, select the link below.\n\n{LINK_TEXT} \n\nThank you."

### Expected result AFTER applying this Pull Request

COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Hello,\n\nA username reminder has been requested for your {SITENAME} account.\n\nYour username is {USERNAME}\n\nTo login to your account, select the link below.\n\n{LINK_TEXT} \n\nThank you."

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 658659403651a263bd8d49c14e8810bf6f389efa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41503">issues.joomla.org/tracker/joomla-cms/41503</a>.</sub>

File: language/en-GB/com_users.ini [hunk 1]
```diff
@@ -21,7 +21,7 @@ COM_USERS_EMAIL_REGISTERED_WITH_ACTIVATION_BODY="Hello {NAME},\n\nThank you for
...
 COM_USERS_EMAIL_REGISTERED_WITH_ADMIN_ACTIVATION_BODY_NOPW="Hello {NAME},\n\nThank you for registering at {SITENAME}. Your account is created and must be verified before you can use it.\n\nTo verify the account select the following link or copy-paste it in your browser:\n{ACTIVATE} \n\nAfter verification an administrator will be notified to activate your account. You'll receive a confirmation when it's done.\n\nOnce that account has been activated you may login to {SITEURL} using the following username and the password you entered during registration:\n\nUsername: {USERNAME}"
-COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Hello,\n\nA username reminder has been requested for your {SITENAME} account.\n\nYour username is {USERNAME}.\n\nTo login to your account, select the link below.\n\n{LINK_TEXT} \n\nThank you."
+COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Hello,\n\nA username reminder has been requested for your {SITENAME} account.\n\nYour username is {USERNAME}\n\nTo login to your account, select the link below.\n\n{LINK_TEXT} \n\nThank you."
```
Comment (Quy) [inline language/en-GB/com_users.ini]: Would `Your username: {USERNAME}`  be better?

Comment (tecpromotion) [inline language/en-GB/com_users.ini]:
```suggestion
COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Hello,\n\nA username reminder has been requested for your {SITENAME} account.\n\nYour username: {USERNAME}\n\nTo login to your account, select the link below.\n\n{LINK_TEXT} \n\nThank you."
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 658659403651a263bd8d49c14e8810bf6f389efa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41503">issues.joomla.org/tracker/joomla-cms/41503</a>.</sub>

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 0e80d7ee32f87dcb1470646f2ccd30f18d2262dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41503">issues.joomla.org/tracker/joomla-cms/41503</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41503">issues.joomla.org/tracker/joomla-cms/41503</a>.</sub>

**Pull request merged**